tensorflow/lite/CMakeLists.txt

```diff
@@ -33,6 +33,14 @@ if(NOT CMAKE_BUILD_TYPE)
   set(CMAKE_BUILD_TYPE "Release")
 endif()
 
+# ccache
```

Comment:
ccache works well with "-DCMAKE_CXX_COMPILER_LAUNCHER=ccache" and users have control.
Wouldn't it preferable?

Reply:
Yes. Thank you.